src/main/resources/sql/company_status/company_status_sql_queries.xml

```diff
@@ -0,0 +1,109 @@
+<sql-statements>
+  <valid-companies-query>
+    <sql-statement>
+SELECT INCORPORATION_NUMBER
```

Comment:
Excellent; but would like code comments for magic numbers and WHERE clauses (e.g. why are we filtering results before 05/01/2010)